pkgs/development/cuda-modules/aliases.nix

```diff
@@ -0,0 +1,4 @@
+# Packges which have been deprecated or removed from cudaPackages
+final: prev: {
+
```

Comment:
We can already move https://github.com/NixOS/nixpkgs/pull/281576/files#diff-cf3f1db251de36cc972110fb95dc6a23bb5ba917c3cc5949069d29ef18c8be05R63-R64 here then

Reply:
not until all references to it have been pruned.